### ISSUE TITLE
bug since 3.3.0 : SystemUnitFamilyType.CreateClassInstance method works uncorrectly 

### ISSUE THREAD

**Issue by x1155665** [AutomationML/AMLEngine#2]
Inheritance relations are ignored when creating new Instance.

example:
SystemUnitClass2 inherits InternalElement1 from SystemUnitClass1.
![testlib](https://cloud.githubusercontent.com/assets/11503677/17398632/03fbaa26-5a3e-11e6-9e35-a838f06d4166.PNG)

Using SystemUnitFamilyType.CreateClassInstance to create an instance of SystemUnitClass2.
In version 3.2.6, the created instance has InternalElement1, while in version 3.3.0 and 3.3.1 it has not.

Code:
```C#
CAEXDocument document1 = CAEXDocument.LoadFromFile(@"./testLib.aml");
SystemUnitFamilyType foundCAEXObject = (SystemUnitFamilyType)document1.CAEXFile.FindFastByPath(@"SystemUnitClassLib1/SystemUnitClass1/SystemUnitClass2");
var newInstance = foundCAEXObject.CreateClassInstance();
document1.CAEXFile.InstanceHierarchy.First.InternalElement.First.Insert_InternalElement((InternalElementType)newInstance);
document1.SaveToFile(@"./output.aml",true);
```

BTW, why is CAEXFile.FindFastByPath() unable to search the instance in instanceHierarchy?

**Comment by AutomationML** [comment]
I will check what's wrong.

InternalElements in InstanceHierarchies are not identifiable by path, because it is not required, that names of siblings are unique.

**Comment by AutomationML** [comment]
Bug fixed in Version 3.3.2
